mbof/views.py

```diff
@@ -40,17 +40,18 @@ class EventListViewSet(generics.ListAPIView):
     serializer_class = EventSerializer
 
     def get_queryset(self):
+        ACTIVE = 'active'
         logger.debug('Type=' + self.kwargs['type'])
         current_time = timezone.now()
         if self.kwargs['type'] is None:
             return Event.objects.all().order_by('postingTime')
         query_type = self.kwargs['type']
         query_type = query_type.replace('/', '')
         if query_type == 'current':
-            return Event.objects.filter(startTime__lte=current_time, endTime__gte=current_time).order_by('postingTime')
+            return Event.objects.filter(startTime__lte=current_time, endTime__gte=current_time, status=ACTIVE).order_by('postingTime')
```

Comment:
Instead of using a local definition of `ACTIVE`, use `Event.STATUS_ACTIVE` instead.